bin/apisix

```diff
@@ -940,6 +940,22 @@ function _M.start(...)
         end
     end
 
+    local argparse = require "argparse"
+    local parser = argparse()
+    parser:argument("_", "Placeholder")
+    parser:option("-c --config", "location of customed config.yaml")
+    local args = parser:parse()
+    local customed_yaml = args["config"]
+
+    local profile = require("apisix.core.profile")
+    profile.apisix_home = apisix_home .. "/"
+    local local_conf_path = profile:yaml_path("config")
+
+    if customed_yaml then
+        local cmd = "cp " .. customed_yaml .. " " .. local_conf_path
```

Comment:
We should allow users to start two APISIX instances, this way is wrong.

Reply:
I got it. I'll change it

Reply:
@membphis If we allow users to start two APISIX instances, users need to specify which conf to use in the following operations after the start step right? Like users could set a name for each instance with specified conf. And when they need to execute certain operation, they could transfer the instance name in headers.

Reply:
we can pass this option by `nginx.conf`. we can follow this way: https://github.com/apache/apisix/blob/master/bin/apisix#L279

Reply:
If users start two APISIX instances, should it create two nginx.conf? If it creates one nginx.conf, we need to read nginx.conf to get the first location of config.yaml, but it seems not so common to read from nginx.conf.

Reply:
Also, I think it would be too complicated when we save the location of customized config.yaml in nginx.conf, since it would be too complicated to pass this one parameter through quite a lot functions to config_local.yaml to read.

Reply:
another question, if the user updates the `config.yaml`, the APISIX instance is no way to load the latest `config.yaml`. 

use `ln` is better than `cp`.

Reply:
Good idea! Update to `ln`